Source/WebCore/platform/graphics/cocoa/FontCacheCoreText.cpp

```diff
@@ -477,15 +477,15 @@ RetainPtr<CTFontRef> preparePlatformFont(CTFontRef originalFont, const FontDescr
     // FIXME: Implement Step 6: the font-variation-settings descriptor inside @font-face
 
     // Step 7: Consult with font-feature-settings inside @font-face
-    if (fontCreationContext.fontFaceFeatures() && !fontCreationContext.fontFaceFeatures()->isEmpty()) {
+    if (fontCreationContext.fontFaceFeatures()) {
```

Comment:
Why?

Reply:
This condition is already handled on the next time with the `for` loop. It seems like a bit of duplication.